sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/systemevents/ResourceDeleteCancelEventData.java

```diff
@@ -234,134 +234,117 @@ public ResourceDeleteCancelEventData setStatus(String status) {
      * Get the authorization property: The requested authorization for the operation.
      *
      * @return the authorization value.
-     * @deprecated This method is no longer supported since v4.9.0.
-     * <p> Use {@link ResourceDeleteCancelEventData#getResourceAuthorization()} instead.
      */
-    @Deprecated
-    public String getAuthorization() {
-        final ResourceAuthorization resourceAuthorization = getResourceAuthorization();
-        try {
-            return DEFAULT_SERIALIZER_ADAPTER.serialize(resourceAuthorization, SerializerEncoding.JSON);
-        } catch (IOException ex) {
-            throw LOGGER.logExceptionAsError(new UncheckedIOException(ex));
-        }
+    public ResourceAuthorization getResourceAuthorization() {
```

Comment:
those changes were made to in https://github.com/Azure/azure-sdk-for-java/commit/6f24cd4199342ed5960e37c3c592e8f2b334fe46#diff-2c74bcf8636d430f253ce43eddb3882318b9e8b424717df6561c4c3235ddcc46R244
 Do we decide to revert it?

Reply:
NVM. It just reorder the methods?

Reply:
Yeah, because autorest.java just appends them to the end of the file.